lib/bot.js

```diff
@@ -372,13 +372,16 @@ module.exports = (function() {
 					return;
 				}
 
+				if(data === null)
+					callback(title+" does not exist", null);
```

Comment:
Please create an instance of `Error` and then pass it to the callback (nodejs convention).